### PR TITLE
Next steps after you import a swagger file in the cli

### PR DESCRIPTION
**Pull request by gratcliff**
1. Updated endpoints to reflect API deprecation of Swagger handlers in main Readme app.
2. Minor logic update to success function to pull relevant data and present updated log string.
3. Updated tests to reflect endpoint change.

### CODE REVIEW

File: lib/swagger.js [hunk 1]
```diff
@@ -28,8 +28,10 @@ exports.run = function({ args, opts }) {
   }
 
   function success(data) {
-    console.log(data);
-    console.log('Success! '.green);
+    const msg = !data.url
+      ? 'Success!'
+      : `You've successfully uploaded a new swagger file to your Readme.io project! You can view it here directly: ${data.url}`;
```
Comment (mjcuva) [inline lib/swagger.js]:
```suggestion
      : `You've successfully uploaded a new swagger file to your ReadMe project! See what was imported here: ${data.url}`;
```

Comment (gratcliff) [reply lib/swagger.js]: Great, thank you @mjcuva!

File: lib/swagger.js [hunk 2]
```diff
@@ -28,8 +28,10 @@ exports.run = function({ args, opts }) {
   }
 
   function success(data) {
-    console.log(data);
-    console.log('Success! '.green);
+    const msg = data.headers.location
+      ? `You've successfully uploaded a new swagger file to your Readme.io project! You can view it here directly: ${data.headers.location}`
```
Comment (mjcuva) [inline lib/swagger.js]: Seems like my copy changes were overwritten!

File: lib/swagger.js [hunk 3]
```diff
@@ -28,8 +28,10 @@ exports.run = function({ args, opts }) {
...
+      : 'Success!';
```
Comment (domharrington) [inline lib/swagger.js]: Do we need this case? Shouldn't the location header always be returned?

File: lib/swagger.js [hunk 4]
```diff
@@ -29,9 +29,16 @@ exports.run = function({ args, opts }) {
 
   function success(data) {
     const msg = data.headers.location
-      ? `You've successfully uploaded a new swagger file to your Readme.io project! You can view it here directly: ${data.headers.location}`
+      ? `You've successfully uploaded a new swagger file to your Readme project! See what was imported here: ${data.headers.location}`
```
Comment (mjcuva) [inline lib/swagger.js]: Small thing, but it should be `ReadMe`!

**Pull request merged**